lib/Dialect/CGGI/IR/CGGIOps.td

```diff
@@ -99,4 +97,23 @@ def CGGI_Lut3Op : CGGI_LutOp<"lut3", [AllTypesMatch<["a", "b", "c", "output"]>]>
   let results = (outs LWECiphertextLike:$output);
 }
 
+def CGGI_LutLinCombOp : CGGI_LutOp<"lut_lincomb", [SameOperandsAndResultType]> {
+  let summary = "A variadic-input lookup table with inputs prepared via linear combination.";
+  let arguments = (ins
+    Variadic<LWECiphertext>:$inputs,
+    DenseI32ArrayAttr:$coefficients,
```

Comment:
The coefficient array size must match the input size, I think. Might also mark that as a TODO in the issue

Reply:
Added in a verifier